xbmc/guilib/GUIBaseContainer.h

```diff
@@ -226,6 +226,9 @@ class CGUIBaseContainer : public IGUIContainer
   std::string m_match;
   float m_scrollItemsPerFrame;
 
+  bool m_waitForScrollEnd = false;
```

Comment:
Can you prefix these lines with a comment "grouping" the variables? For example, see the two groups at https://github.com/xbmc/xbmc/blob/master/xbmc/input/joysticks/generic/FeatureHandling.h#L129.